src/Space.cpp

```diff
@@ -384,8 +384,8 @@ void Space::GetHyperspaceExitParams(const SystemPath &source, const SystemPath &
 		(max_orbit_vel * max_orbit_vel);
 	dist = std::max(dist, primary->GetSystemBody()->GetRadius() * 10);
 
-	// ensure an absolut minimum distance
-	dist = std::max(dist, 0.2 * AU);
+	// ensure an absolut minimum and an absolut maximum distance
+	dist = Clamp(dist, 0.2 * AU, 100 * AU);
```

Comment:
Maybe change this too `dist = Clamp(dist, 0.2 * AU, std::max(primary->GetSystemBody()->GetRadius() * 1.1, 100 * AU));` to handle stars that are larger than 100 AU